lib/virt_autotest/utils.pm

```diff
@@ -772,12 +772,10 @@ sub is_sev_es_guest {
     $guest_name //= '';
     croak('Arugment guest_name should not be empty') if ($guest_name eq '');
 
-    $guest_name =~ /(sev-es|sev)/img;
-    if ($1 ne '') {
+    if ($guest_name =~ /(sev-es|sev)/img) {
```

Comment:
Do you really need `Global` and `Multiline` flags?

Reply:
imho, there is not really need `Global` and `Multiline` flags
Just only keep the current code format.